src/main.cc

```diff
@@ -114,6 +114,12 @@ jint JNI_OnLoad(JavaVM* vm, void* reserved)
 
 int main(int argc, char *argv[])
 {
+#ifdef Q_OS_UNIX
+    //FOrce writing to the console on UNIX/BSD devices
```

Comment:
FAcepalm. @DonLakeFlyer  if this passes CI do you mind fixing this comment before merging to master? I hate triggering another CI build to fix one incorrectly cased letter.